docs/user/02-configuration.md

```diff
@@ -28,46 +28,49 @@ Use the following sample CRs as guidance. Each can be applied immediately when y
-| **publisher.&#x200b;resources.&#x200b;limits**  | map\[string\]\{integer or string\} | Limits describes the maximum amount of compute resources allowed. More info: https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/                                                                                                                                                                                |
+| Parameter | Type | Description |
+| ---- | ----------- | ---- |
+| **annotations**  | map\[string\]string | Annotations allows to add annotations to resources. |
+| **backend**  | object | Backend defines the active backend used by Eventing. |
+| **backend.&#x200b;config**  | object | Config defines configuration for eventing backend. |
+| **backend.&#x200b;config.&#x200b;domain**  | string | Domain defines the cluster public domain used to configure the EventMesh Subscriptions and their corresponding ApiRules. |
+| **backend.&#x200b;config.&#x200b;eventMeshSecret**  | string | EventMeshSecret defines the namespaced name of K8s Secret containing EventMesh credentials. The format of name is "namespace/name". |
+| **backend.&#x200b;config.&#x200b;eventTypePrefix**  | string |  |
+| **backend.&#x200b;config.&#x200b;natsMaxMsgsPerTopic**  | integer | NATSMaxMsgsPerTopic limits how many messages in the NATS stream to retain per subject. |
+| **backend.&#x200b;config.&#x200b;natsStreamMaxSize**  | \{integer or string\} | NATSStreamMaxSize defines the maximum storage size for stream data. |
+| **backend.&#x200b;config.&#x200b;natsStreamReplicas**  | integer | NATSStreamReplicas defines the number of replicas for stream. |
+| **backend.&#x200b;config.&#x200b;natsStreamStorageType**  | string | NATSStreamStorageType defines the storage type for stream data. |
+| **backend.&#x200b;type** (required) | string | Type defines which backend to use. The value is either `EventMesh`, or `NATS`. |
+| **labels**  | map\[string\]string | Labels allows to add Labels to resources. |
+| **logging**  | object | Logging defines the log level for eventing-manager. |
+| **logging.&#x200b;logLevel**  | string | LogLevel defines the log level. |
+| **publisher**  | object | Publisher defines the configurations for eventing-publisher-proxy. |
+| **publisher.&#x200b;replicas**  | object | Replicas defines the scaling min/max for eventing-publisher-proxy. |
+| **publisher.&#x200b;replicas.&#x200b;max**  | integer | Max defines maximum number of replicas. |
+| **publisher.&#x200b;replicas.&#x200b;min**  | integer | Min defines minimum number of replicas. |
+| **publisher.&#x200b;resources**  | object | Resources defines resources for eventing-publisher-proxy. |
+| **publisher.&#x200b;resources.&#x200b;claims**  | \[\]object | Claims lists the names of resources, defined in spec.resourceClaims, that are used by this container. 
```

Comment:
This isn't rendered correctly when the sentences are in different lines:
<img width="1026" alt="image" src="https://github.com/kyma-project/eventing-manager/assets/76950046/44eb8a6d-2c5b-4607-bc9c-b7fdfc6c4a13">

The table is auto-generated, so you might have to adjust the formatting somewhere else instead of here. But here's how it should look like eventually:

```suggestion
| **publisher.&#x200b;resources.&#x200b;claims**  | \[\]object | Claims lists the names of resources, defined in spec.resourceClaims, that are used by this container. This is an alpha field and requires enabling the DynamicResourceAllocation feature gate. This field is immutable. It can only be set for containers. |
```